extensions/parquet/table/src/main/java/io/deephaven/parquet/table/ParquetTools.java

```diff
@@ -1300,13 +1489,20 @@ public static Table readSingleFileTable(
      * (by {@link ParquetTableLocationKey location key} order) location found, which must have non-null partition values
      * for all partition keys.
      *
+     * <p>
+     * Callers may prefer the simpler methods {@link #readTable(String, ParquetInstructions)} with layout provided using
+     * {@link ParquetInstructions.Builder#setFileLayout(ParquetInstructions.ParquetFileLayout)}.
+     *
      * @param locationKeyFinder The source of {@link ParquetTableLocationKey location keys} to include
      * @param readInstructions Instructions for customizations while reading
      * @return The table
      */
     public static Table readPartitionedTable(
```

Comment:
Can these methods which take TableLocationKeyFinder as argument be made private?
There's one such overload of `readSingleFileTable` and a few overloads of `readPartitionedTable` that fall in this category.